app/models/prx/api.rb

```diff
@@ -0,0 +1,144 @@
+module Prx
+  module Api
+    class PrxHyperResource < HyperResource
```

Comment:
Also, we never used the prx_access gem in this repo (which I'm pretty sure is where it started life?)  Maybe we'll port it back out at some point.

Reply:
yeah, this PR has enough refactoring ;)

Reply:
added #1207  for this